### DISCUSSION TITLE
HTML files

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#628] [Content] [answered]
I'm somewhat new to this project and I was wondering if there is a purpose for keeping some files in HTML format?

**Comment by hamishwillee** [comment]
Originally some parts were maintained by different teams, but I think they are all gone. 

@wbamberg This is the list of unconverted HTML - would these all be accidental omissions?

en-us\mozilla\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_alertdialog_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_article_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_group_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_link_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_log_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_presentation_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_progressbar_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_radio_role\index.htmlen-us\web\accessibility\aria\aria_techniques\using_the_slider_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_status_role\index.html
en-us\web\accessibility\aria\aria_techniques\using_the_toolbar_role\index.html
en-us\web\accessibility\aria\aria_techniques\x-ms-aria-flowfrom\index.html
en-us\web\accessibility\aria\aria_test_cases\index.html
en-us\web\accessibility\aria\forms\alerts\index.html
en-us\web\accessibility\aria\forms\basic_form_hints\index.html
en-us\web\accessibility\aria\forms\multipart_labels\index.html
en-us\web\accessibility\aria\web_applications_and_aria_faq\index.html
en-us\web\accessibility\aria\widgets\index.html
en-us\web\accessibility\html_to_msaa\index.html
en-us\web\accessibility\implementing_msaa_server\index.html
en-us\web\accessibility\understanding_wcag\perceivable\color_contrast\index.html
en-us\web\accessibility\understanding_wcag\perceivable\use_of_color\index.html

**Reply by wbamberg** [reply]
These are files that Estelle was working on, and wanted to keep as HTML until the project was done. @estelle , do you know if we can remove these now?

**Reply by MrBrain295** [reply]
Some of these are fixed by: #15139.

**Reply by wbamberg** [reply]
@estelle , these are still in HTML:

./files/en-us/web/accessibility/implementing_msaa_server/index.html
./files/en-us/web/accessibility/html_to_msaa/index.html
./files/en-us/web/accessibility/aria/aria_test_cases/index.html
./files/en-us/web/accessibility/aria/forms/multipart_labels/index.html
./files/en-us/web/accessibility/aria/forms/basic_form_hints/index.html
./files/en-us/web/accessibility/aria/forms/alerts/index.html
./files/en-us/web/accessibility/aria/web_applications_and_aria_faq/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_presentation_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/x-ms-aria-flowfrom/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_toolbar_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_status_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_group_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_link_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_log_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_alertdialog_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_slider_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_progressbar_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_article_role/index.html
./files/en-us/web/accessibility/aria/widgets/index.html
./files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.html
./files/en-us/web/accessibility/understanding_wcag/perceivable/use_of_color/index.html

Are you still working on them? Would be great to get to 0 HTML files. There's also:

en-us\mozilla\index.html

...which I will take care of now.

**Reply by wbamberg** [reply]
-> https://github.com/mdn/content/pull/15259

**Reply by Rumyra** [reply]
@wbamberg - thank you so much ❤️

**Reply by estelle** [reply]
half of those were deleted and redirected. 

These should be gone:
/files/en-us/web/accessibility/aria/aria_techniques/using_the_toolbar_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_status_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_group_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_link_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_log_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_alertdialog_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_slider_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_progressbar_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_article_role/index.html
./files/en-us/web/accessibility/aria/widgets/index.html
./files/en-

**Comment by MrBrain295** [comment, marked as answer]
Moved to mdn/mdn-community#99.

**Comment by estelle** [comment]
Of the above, all should be deleted except for 
./files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.html
./files/en-us/web/accessibility/understanding_wcag/perceivable/use_of_color/index.html
